lib/SGN/Controller/AJAX/BreedersToolbox/GraphicalFilteringAJAX.pm

```diff
@@ -0,0 +1,215 @@
+use strict;
+
+package SGN::Controller::AJAX::BreedersToolbox::GraphicalFiltering;
+
+use Data::Dumper;
+use Moose;
+use Cwd qw(cwd);
+use CXGN::List;
+use CXGN::BreederSearch;
+use CXGN::Phenotypes::PhenotypeMatrix;
+
+BEGIN { extends 'Catalyst::Controller::REST' }
+
+__PACKAGE__->config(
+    default   => 'application/json',
+    stash_key => 'rest',
+    map       => { 'application/json' => 'JSON', 'text/html' => 'JSON' },
+   );
+
+
+has 'schema' => (
+		 is       => 'rw',
+		 isa      => 'DBIx::Class::Schema',
+		 lazy_build => 1,
+		);
+
+sub common_traits_by_trial_list : Path('/ajax/plot/common_traits_by/trial_list') : ActionClass('REST') {}
+sub common_traits_by_trial_list_GET : Args(0) {
+   my $self = shift;
+   my $c = shift;
+
+   #get list ID from url param
+   my $trial_list_id = $c->request->param('trial_list_id');
+
+   #get userinfo from db
+   my $schema = $c->dbic_schema("Bio::Chado::Schema");
+   my $user = $c->user();
```

Comment:
- [x] [added check and alert if user is not logged in upon plot phenotypes AJAX call](https://github.com/solgenomics/sgn/pull/1232/commits/ce2ae21c95a88eab35f87ae8af51db33d21043cf)